src/data/events.json

```diff
@@ -1,4 +1,15 @@
 [
+    {
+    "eventName": "Market Research Workshop for Startup Founders & Tech Product Showcase",
+    "eventDescription": "A gathering of founders and startup folks from Chennai!",
+    "eventDate": "2025-06-28",
+    "eventTime": "09:00",
+    "eventVenue": "YuniQ, 4th Floor, TICEL BIOPARK, 5, CSIR Rd, Phase II, Tharamani, Chennai, Tamil Nadu 600113, India",
+    "eventLink": "https://allevents.in/chennai/market-research-workshop-for-startup-founders-and-tech-product-showcase-tickets/80002194804994?utm_source=Tamilnadu+Tech&utm_medium=website&utm_campaign=Tamilnadu+Tech&ref=utm-builder",
+    "location": "Chennai",
+    "communityName": "The Startup Club",
+    "communityLogo": "https://thestartupclub.info/images/startup-club-logo.png"
```

Comment:
hey @ryarasi 
thanks a lot for contributing but it looks like you have not used imgbb, since the image cdn is not in our next.config files it wont be able to fetch the image, so It would be great if you could change it to imgbb embeddable link and then update the PR.

we would be more than happy to merge the changes. 

Cheers :)